### PR TITLE
KIALI-2361 Make graph side panel collapsible

### PR DESCRIPTION
**Pull request by israel-hdez**
https://issues.jboss.org/browse/KIALI-2361

peek 2019-02-06 17-16

### CODE REVIEW

File: src/pages/Graph/SummaryPanelEdge.tsx [hunk 1]
```diff
@@ -37,9 +37,11 @@ type SummaryPanelEdgeState = {
 
 export default class SummaryPanelEdge extends React.Component<SummaryPanelPropType, SummaryPanelEdgeState> {
   static readonly panelStyle = {
-    width: '25em',
+    height: '100%',
+    margin: 0,
     minWidth: '25em',
-    overflowY: 'auto' as 'auto'
+    overflowY: 'auto' as 'auto',
```
Comment (mtho11) [inline src/pages/Graph/SummaryPanelEdge.tsx]:
```
   'auto' as 'auto',
```
I don't understand why adding the `as 'auto'` part?

File: src/pages/Overview/OverviewStatuses.tsx [hunk 2]
```diff
@@ -33,9 +33,7 @@ class OverviewStatuses extends React.Component<Props> {
     }
     return (
       <>
-        <Link to={`/${targetPage}?namespaces=${name}`} >
-          {text}
-        </Link>
+        <Link to={`/${targetPage}?namespaces=${name}`}>{text}</Link>
```
Comment (mtho11) [inline src/pages/Overview/OverviewStatuses.tsx]: This is probably not relevant to this PR.

**Comment by mtho11** [comment]
I guess we need to wait on this until UxD approves

File: src/pages/Graph/SummaryPanelEdge.tsx (hunk 1 above)
Comment (israel-hdez) [reply src/pages/Graph/SummaryPanelEdge.tsx]: There is a compile error if it's not casted.

File: src/components/SummaryPanel/RateChart.tsx [hunk 3]
```diff
@@ -75,24 +77,26 @@ export class RateChartGrpc extends React.Component<RateChartGrpcPropType> {
 }
 
 type RateChartHttpPropType = {
+  height?: number;
+  legendPos?: string; // e.g. right, left
```
[the remark this answers is not in the document]
Comment (israel-hdez) [reply src/components/SummaryPanel/RateChart.tsx]: Done!

File: src/pages/Overview/OverviewStatuses.tsx (hunk 2 above)
Comment (israel-hdez) [reply src/pages/Overview/OverviewStatuses.tsx]: It went after rebasing. This was changed in another PR.

**Comment by israel-hdez** [comment]
@jshaughn I fixed the tour placement, but now it's displayed on top of the graph:

image

I couldn't find a better way to fix it.

**Comment by mtho11** [comment]
Cool, @jshaughn your label change is in.

**Pull request merged**